package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "fsh-ihsus",
-  "version": "0.0.1",
-  "description": "Ichthymorphic Helper Systematically Undoes Sushi (FSH Decompiler)",
+  "name": "gofsh",
+  "version": "0.1.0",
+  "description": "GoFSH is a FHIR Shorthand (FSH) decompiler, able to convert formal FHIR definitions from JSON/XML to FSH.",
```

Comment:
I don't think we can yet claim XML to FSH.

Reply:
I think I was thinking of the description more as the target than the current reality.  But point taken.

Reply:
Addressed in 53dc60d